compiler/src/iree/compiler/Codegen/SPIRV/SPIRVTile.cpp

```diff
@@ -45,46 +45,68 @@ namespace iree_compiler {
 
 /// Collects computation ops which we will use as anchor to tile and fuse.
 static LogicalResult collectComputeOps(
-    func::FuncOp funcOp, SmallVectorImpl<Operation *> &computeOps) {
+    func::FuncOp funcOp, SmallVectorImpl<Operation *> &computeOps,
```

Comment:
Maybe return `FailureOr<LoweringConfigAttr>` to avoid return by reference?

Reply:
Done.